cli/tests/cmd.rs

```diff
@@ -256,7 +256,7 @@ forgetest_ignore!(can_compile_local_spells, |_: TestProject, mut cmd: TestComman
     let dss_exec_lib = "src/DssSpell.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4";
 
     cmd.args([
-        "build",
+        "test",
```

Comment:
Unneeded?

Reply:
yeah this function is ignored, but kept in so it can be easily used when running manual tests against a project locally, like `spells`.
will roll back the changes there

Reply:
Ok makes sense